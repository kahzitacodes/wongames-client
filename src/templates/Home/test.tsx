import { screen } from '@testing-library/react'

import { Home } from '.'
import { renderWithTheme } from '@/utils/tests/helpers'

describe('<Home />', () => {
  it('should render menu and footer', () => {
    const { container } = renderWithTheme(<Home />)

    expect(screen.getByLabelText(/open menu/i)).toBeInTheDocument()

    expect(
      screen.getByRole('heading', { name: /contact/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })

  it('should render sections', () => {
    renderWithTheme(<Home />)

    expect(screen.getByRole('heading', { name: /news/i })).toBeInTheDocument()
    expect(
      screen.getByRole('heading', { name: /most popular/i })
    ).toBeInTheDocument()
    expect(
      screen.getByRole('heading', { name: /upcoming/i })
    ).toBeInTheDocument()
    expect(
      screen.getByRole('heading', { name: /free games/i })
    ).toBeInTheDocument()
  })
})
