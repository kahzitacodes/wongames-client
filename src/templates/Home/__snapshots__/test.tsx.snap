// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render menu and footer 1`] = `
.c0 {
  max-width: 130rem;
  margin-left: auto;
  margin-right: auto;
  padding-left: calc(3.2rem / 2);
  padding-right: calc(3.2rem / 2);
}

.c1 {
  display: flex;
  align-items: center;
  padding: 1.2rem 0;
  position: relative;
}

.c12 {
  display: flex;
  gap: 0.8rem;
  align-items: center;
  flex: 1;
  justify-content: flex-end;
}

.c4 {
  color: #FAFAFA;
  cursor: pointer;
  width: 2.8rem;
  height: 2.8rem;
  padding: 0.4rem;
  border-radius: 0.4rem;
}

.c4 svg {
  width: 2rem;
  height: 2rem;
}

.c4:hover {
  background-color: #2C2E3E;
}

.c9 {
  margin-left: 2.4rem;
  gap: 2.4rem;
}

.c11 {
  color: #FAFAFA;
  position: relative;
  font-size: 1.6rem;
  margin: 0.3rem 2.4rem 0;
  text-decoration: none;
  text-align: center;
}

.c11:hover::after {
  content: '';
  position: absolute;
  height: 0.3rem;
  bottom: -0.4rem;
  background-color: #F366A7;
  animation: hoverAnimation 0.2s forwards;
}

.c14 {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  background: #FAFAFA;
  position: absolute;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  height: 100vh;
  overflow: hidden;
  opacity: 0;
  padding: 4.0rem;
  pointer-events: none;
}

.c14 .c3 {
  position: absolute;
  right: 2.4rem;
  top: 2.4rem;
  color: #6F7287;
}

.c14 .c3:hover {
  background-color: #ECECEF;
}

.c14 .c10 {
  color: #05061A;
  font-weight: 600;
  font-size: 2.0rem;
  transform: translateY(3rem);
  transition: transform 0.3s ease-in-out;
}

.c14 .c8 {
  display: flex;
  flex-direction: column;
  flex: 1;
  justify-content: center;
  align-items: center;
}

.c14 .c15 {
  transform: translateY(3rem);
  transition: transform 0.3s ease-in-out;
}

.c16 {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.c16 >span {
  display: block;
  margin: 0.8rem 0;
  font-size: 1.2rem;
}

.c18 {
  text-decoration: none;
  color: #F366A7;
  border-bottom: 0.2rem solid #F366A7;
  font-weight: 600;
}

.c6 {
  color: #FAFAFA;
  width: 11rem;
  height: 3.3rem;
}

.c23 {
  color: #05061A;
  width: 11rem;
  height: 3.3rem;
}

.c13 {
  background: linear-gradient( 180deg,#F366A7 0%,#DC448A 40% );
  color: #FAFAFA;
  border: none;
  border-radius: 0.4rem;
  font-weight: 600;
  line-height: 0%;
  display: flex;
  align-items: center;
  cursor: pointer;
  gap: 0.8rem;
  background-size: 200% 200%;
  transition: background-position 0.2s linear;
  text-decoration: none;
  height: 4rem;
  font-size: 1.4rem;
  padding: 0 1.6rem;
  width: fit-content;
}

.c13:hover {
  background-position: 0 100%;
}

.c13 svg {
  width: 2rem;
}

.c17 {
  background: linear-gradient( 180deg,#F366A7 0%,#DC448A 40% );
  color: #FAFAFA;
  border: none;
  border-radius: 0.4rem;
  font-weight: 600;
  line-height: 0%;
  display: flex;
  align-items: center;
  cursor: pointer;
  gap: 0.8rem;
  background-size: 200% 200%;
  transition: background-position 0.2s linear;
  text-decoration: none;
  height: 4.8rem;
  font-size: 1.6rem;
  padding: 0 2.4rem;
  width: fit-content;
  width: 100%;
  justify-content: center;
}

.c17:hover {
  background-position: 0 100%;
}

.c17 svg {
  width: 2.4rem;
}

.c2 {
  display: none;
}

.c7 {
  display: none;
}

.c20 {
  line-height: 130%;
  color: #05061A;
  line-height: 130%;
  padding-left: 1.6rem;
  border-left: 0.6rem solid #2496FF;
  font-size: 2.0rem;
}

.c21 {
  line-height: 130%;
  color: #FAFAFA;
  line-height: 130%;
  padding-left: 1.6rem;
  border-left: 0.6rem solid #2496FF;
  font-size: 2.0rem;
}

.c26 {
  line-height: 130%;
  color: #05061A;
  line-height: 130%;
  position: relative;
  margin-bottom: 3.2rem;
  font-size: 1.6rem;
}

.c26::after {
  content: '';
  position: absolute;
  left: 0;
  bottom: -1rem;
  width: 5rem;
  border-bottom: 0.6rem solid #F366A7;
}

.c26::after {
  border-bottom-width: 0.4rem;
  width: 3rem;
}

.c22 {
  padding: 2.4rem 0;
}

.c22 .c19 {
  text-transform: uppercase;
}

.c24 {
  display: grid;
  grid-template-columns: repeat(2,1fr);
  margin-top: 3.2rem;
  gap: 2.4rem;
}

.c25 {
  display: flex;
  flex-direction: column;
}

.c25 a,
.c25 span {
  color: #6F7287;
  text-decoration: none;
  font-size: 1.4rem;
  display: block;
}

.c25 a:hover {
  text-decoration: decoration;
}

.c27 {
  text-align: center;
  color: #6F7287;
  font-size: 1.2rem;
  margin: 3.2rem 0 0;
}

@media (max-width: 768px) {
  .c5 {
    position: absolute;
    left: 50%;
    transform: translateX(-50%);
  }
}

@media (min-width: 768px) {
  .c12 {
    gap: 1.6rem;
  }
}

@media (max-width: 768px) {
  .c6 {
    width: 5.8rem;
    height: 4.0rem;
  }

  .c6 svg {
    pointer-events: none;
    height: 4.0rem;
  }

  .c6 .text {
    display: none;
  }
}

@media (max-width: 768px) {
  .c2 {
    display: block;
  }
}

@media (min-width: 768px) {
  .c7 {
    display: block;
  }
}

@media (min-width: 768px) {
  .c20 {
    font-size: 2.8rem;
  }
}

@media (min-width: 768px) {
  .c21 {
    font-size: 2.8rem;
  }
}

@media (min-width: 768px) {
  .c24 {
    grid-template-columns: repeat(4, 1fr);
    gap: 3.2rem;
  }
}

<section>
  <div
    class="c0"
  >
    <menu
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          aria-label="open menu"
          class="c3 c4"
        >
          <svg
            class="lucide lucide-menu"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="4"
              x2="20"
              y1="12"
              y2="12"
            />
            <line
              x1="4"
              x2="20"
              y1="6"
              y2="6"
            />
            <line
              x1="4"
              x2="20"
              y1="18"
              y2="18"
            />
          </svg>
        </div>
      </div>
      <div
        class="c5"
      >
        <div
          class="c6"
          color="white"
        >
          <svg
            aria-label="Won Games"
            fill="none"
            role="img"
            viewBox="0 0 158 48"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M.055 15.227.019 35.872c-.008 4.796 5.336 7.663 9.327 5.004l20.596-13.722L50.49 40.947c3.982 2.672 9.335-.176 9.344-4.972l.034-19.997c.014-7.882-7.549-13.565-15.116-11.361l-.873.254a50.875 50.875 0 0 1-29.358-.27C7.351 2.366.068 7.716.055 15.226Z"
              fill="#F366A7"
            />
            <path
              d="m30.163 27.16-6.033-4.292a.5.5 0 0 0-.782.494l1.293 7.298 5.523-3.5Z"
              fill="#110F28"
              fill-opacity=".15"
            />
            <path
              d="M7.592 7.103c-5.11 2.755-4.519 5-4.519 7.5M57.274 26.103v1M3.052 18.103v2M57.274 30.103v5"
              stroke="#fff"
              stroke-linecap="round"
              stroke-opacity=".4"
              stroke-width="1.5"
            />
            <path
              d="M13.053 15.103v6M10.04 18.103h6.025"
              stroke="#0A0D27"
              stroke-linecap="round"
              stroke-width="2"
            />
            <ellipse
              cx="1.506"
              cy="1.5"
              fill="#0A0D27"
              rx="1.506"
              ry="1.5"
              transform="matrix(1 0 0 1 43.217 12.16)"
            />
            <ellipse
              cx="1.506"
              cy="1.5"
              fill="#0A0D27"
              rx="1.506"
              ry="1.5"
              transform="matrix(1 0 0 1 48.237 16.103)"
            />
            <ellipse
              cx="1.506"
              cy="1.5"
              fill="#0A0D27"
              rx="1.506"
              ry="1.5"
              transform="matrix(1 0 0 1 43.217 20.103)"
            />
            <ellipse
              cx="1.506"
              cy="1.5"
              fill="#0A0D27"
              rx="1.506"
              ry="1.5"
              transform="matrix(1 0 0 1 38.196 16.103)"
            />
            <path
              class="text"
              d="M79.263 12.633h-.06c-.68.1-1.2.44-1.56 1.02-.22.38-.33.78-.33 1.2 0 .1.01.2.03.3v.03l1.38 9.33c.08.54.34.99.78 1.35a2.27 2.27 0 0 0 1.77.45c.66-.1 1.17-.42 1.53-.96.26-.36.39-.77.39-1.23 0-.08-.01-.17-.03-.27l-1.38-9.33c-.08-.56-.34-1.03-.78-1.41-.42-.34-.9-.51-1.44-.51-.1 0-.2.01-.3.03Zm18.78-.03c-.54 0-1.02.18-1.44.54-.44.36-.7.82-.78 1.38l-1.41 9.51c-.02.08-.03.15-.03.21 0 .52.17.98.51 1.38.36.46.84.73 1.44.81.1.02.21.03.33.03.66 0 1.21-.24 1.65-.72.3-.32.49-.7.57-1.14v-.06l1.38-9.36v-.03c.02-.1.03-.2.03-.3 0-.42-.12-.82-.36-1.2-.36-.58-.87-.92-1.53-1.02h-.06c-.1-.02-.2-.03-.3-.03Zm-9.24 8.04h-.06c-.08 0-.16.01-.24.03-.36.04-.69.16-.99.36-.16.1-.3.22-.42.36-.08.08-.21.22-.39.42l-2.19 2.7c-.18.2-.42.48-.72.84-.28.34-.51.59-.69.75-.38.34-.75.58-1.11.72-.2.08-.41.14-.63.18-.14.02-.29.03-.45.03-.38 0-.74-.07-1.08-.21-.32-.12-.6-.29-.84-.51l.75 5.13v.06c.02.04.03.09.03.15.04.18.09.35.15.51.14.3.35.56.63.78.18.16.38.28.6.36.24.08.49.12.75.12h.09c.16 0 .3-.01.42-.03.46-.1.85-.32 1.17-.66l.3-.3 4.89-6.03 4.74 5.82c.24.28.39.45.45.51.38.38.83.61 1.35.69h.33c.28 0 .55-.05.81-.15.42-.18.77-.46 1.05-.84.18-.28.3-.59.36-.93v-.06l.75-5.01c-.34.3-.73.5-1.17.6-.26.08-.52.12-.78.12a2.903 2.903 0 0 1-1.26-.27c-.26-.12-.59-.37-.99-.75-.26-.26-.59-.65-.99-1.17l-2.52-3.06c-.14-.18-.28-.34-.42-.48-.2-.2-.39-.35-.57-.45-.3-.2-.64-.31-1.02-.33h-.09Zm19.806.3c.28-.26.6-.46.96-.6.34-.14.69-.21 1.05-.21.46 0 .9.11 1.32.33.48.24.87.58 1.17 1.02a5.912 5.912 0 0 1 2.13-3.15 5.73 5.73 0 0 1 3.6-1.23c1.66 0 3.07.59 4.23 1.77 1.16 1.16 1.74 2.57 1.74 4.23 0 1.66-.58 3.08-1.74 4.26-1.16 1.16-2.57 1.74-4.23 1.74-1.34 0-2.55-.4-3.63-1.2a5.913 5.913 0 0 1-2.13-3.18c-.22.34-.5.63-.84.87-.5.3-1.04.45-1.62.45-.42 0-.83-.08-1.23-.24-.3-.14-.56-.32-.78-.54.5 2.42 1.71 4.42 3.63 6 1.92 1.56 4.12 2.34 6.6 2.34 1.9 0 3.65-.47 5.25-1.41 1.62-.94 2.89-2.21 3.81-3.81.94-1.62 1.41-3.38 1.41-5.28 0-1.9-.47-3.65-1.41-5.25-.92-1.62-2.19-2.9-3.81-3.84-1.6-.94-3.35-1.41-5.25-1.41-2.46 0-4.65.79-6.57 2.37-1.92 1.56-3.14 3.55-3.66 5.97Zm2.01-.09c-.66 0-1.21.24-1.65.72-.24.24-.41.53-.51.87-.06.22-.09.45-.09.69 0 .22.03.43.09.63.08.28.21.53.39.75.44.56 1.03.84 1.77.84.72 0 1.3-.27 1.74-.81.22-.26.37-.54.45-.84.04-.2.06-.4.06-.6 0-.18-.02-.35-.06-.51-.08-.38-.24-.71-.48-.99a2.23 2.23 0 0 0-1.71-.75Zm29.649-8.25c-.6 0-1.12.21-1.56.63-.46.44-.69.98-.69 1.62v6.15c.24-.28.53-.5.87-.66.42-.2.87-.3 1.35-.3h.06c.46 0 .9.1 1.32.3a2.909 2.909 0 0 1 1.05.84l9.81 11.55c.44.58 1.03.87 1.77.87.66 0 1.22-.25 1.68-.75.38-.42.57-.92.57-1.5v-6.12c-.26.26-.56.47-.9.63-.42.2-.87.3-1.35.3-.56 0-1.08-.13-1.56-.39-.34-.2-.63-.45-.87-.75l-9.87-11.67c-.22-.24-.48-.43-.78-.57a2.26 2.26 0 0 0-.9-.18Zm13.98 0c-.62 0-1.14.21-1.56.63-.46.44-.69.98-.69 1.62v8.7c0 .42.14.8.42 1.14.44.56 1.05.84 1.83.84.54 0 1.02-.15 1.44-.45.52-.4.79-.91.81-1.53v-8.7c-.02-.7-.29-1.27-.81-1.71-.42-.36-.9-.54-1.44-.54Zm-14.01 8.07c-.6 0-1.11.19-1.53.57-.44.4-.67.88-.69 1.44v8.67c0 .5.15.95.45 1.35.46.6 1.06.9 1.8.9.64 0 1.18-.23 1.62-.69.42-.42.63-.94.63-1.56v-8.67c-.02-.8-.41-1.39-1.17-1.77-.32-.16-.67-.24-1.05-.24h-.06Z"
              fill="currentColor"
            />
          </svg>
        </div>
      </div>
      <div
        class="c7"
      >
        <div
          class="c8 c9"
        >
          <a
            class="c10 c11"
            href="#"
          >
            Home
          </a>
          <a
            class="c10 c11"
            href="#"
          >
            Explore
          </a>
        </div>
      </div>
      <div
        class="c12"
      >
        <div
          aria-label="search"
          class="c3 c4"
        >
          <svg
            class="lucide lucide-search"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="11"
              cy="11"
              r="8"
            />
            <path
              d="m21 21-4.3-4.3"
            />
          </svg>
        </div>
        <div
          aria-label="open shopping cart"
          class="c3 c4"
        >
          <svg
            class="lucide lucide-shopping-cart"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="8"
              cy="21"
              r="1"
            />
            <circle
              cx="19"
              cy="21"
              r="1"
            />
            <path
              d="M2.05 2.05h2l2.66 12.42a2 2 0 0 0 2 1.58h9.78a2 2 0 0 0 1.95-1.57l1.65-7.43H5.12"
            />
          </svg>
        </div>
        <div
          class="c7"
        >
          <button
            class="c13"
          >
            <span>
              Sign in
            </span>
          </button>
        </div>
      </div>
      <nav
        aria-hidden="true"
        class="c14"
      >
        <div
          aria-label="close menu"
          class="c3 c4"
        >
          <svg
            class="lucide lucide-x"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M18 6 6 18"
            />
            <path
              d="m6 6 12 12"
            />
          </svg>
        </div>
        <div
          class="c8 c9"
        >
          <a
            class="c10 c11"
            href="#"
          >
            Home
          </a>
          <a
            class="c10 c11"
            href="#"
          >
            Explore
          </a>
        </div>
        <div
          class="c15 c16"
        >
          <button
            class="c17"
          >
            <span>
              Log in now
            </span>
          </button>
          <span>
            or
          </span>
          <a
            class="c18"
          >
            Create account
          </a>
        </div>
      </nav>
    </menu>
  </div>
  <div
    class="c0"
  >
    <h2
      class="c19 c20"
      color="black"
    >
      News
    </h2>
  </div>
  <div
    class="c0"
  >
    <h2
      class="c19 c21"
      color="white"
    >
      Most popular
    </h2>
  </div>
  <div
    class="c0"
  >
    <h2
      class="c19 c21"
      color="white"
    >
      Upcoming
    </h2>
  </div>
  <div
    class="c0"
  >
    <h2
      class="c19 c21"
      color="white"
    >
      Free games
    </h2>
  </div>
  <div
    class="c0"
  >
    <footer
      class="c22"
    >
      <div
        class="c23"
        color="black"
      >
        <svg
          aria-label="Won Games"
          fill="none"
          role="img"
          viewBox="0 0 158 48"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M.055 15.227.019 35.872c-.008 4.796 5.336 7.663 9.327 5.004l20.596-13.722L50.49 40.947c3.982 2.672 9.335-.176 9.344-4.972l.034-19.997c.014-7.882-7.549-13.565-15.116-11.361l-.873.254a50.875 50.875 0 0 1-29.358-.27C7.351 2.366.068 7.716.055 15.226Z"
            fill="#F366A7"
          />
          <path
            d="m30.163 27.16-6.033-4.292a.5.5 0 0 0-.782.494l1.293 7.298 5.523-3.5Z"
            fill="#110F28"
            fill-opacity=".15"
          />
          <path
            d="M7.592 7.103c-5.11 2.755-4.519 5-4.519 7.5M57.274 26.103v1M3.052 18.103v2M57.274 30.103v5"
            stroke="#fff"
            stroke-linecap="round"
            stroke-opacity=".4"
            stroke-width="1.5"
          />
          <path
            d="M13.053 15.103v6M10.04 18.103h6.025"
            stroke="#0A0D27"
            stroke-linecap="round"
            stroke-width="2"
          />
          <ellipse
            cx="1.506"
            cy="1.5"
            fill="#0A0D27"
            rx="1.506"
            ry="1.5"
            transform="matrix(1 0 0 1 43.217 12.16)"
          />
          <ellipse
            cx="1.506"
            cy="1.5"
            fill="#0A0D27"
            rx="1.506"
            ry="1.5"
            transform="matrix(1 0 0 1 48.237 16.103)"
          />
          <ellipse
            cx="1.506"
            cy="1.5"
            fill="#0A0D27"
            rx="1.506"
            ry="1.5"
            transform="matrix(1 0 0 1 43.217 20.103)"
          />
          <ellipse
            cx="1.506"
            cy="1.5"
            fill="#0A0D27"
            rx="1.506"
            ry="1.5"
            transform="matrix(1 0 0 1 38.196 16.103)"
          />
          <path
            class="text"
            d="M79.263 12.633h-.06c-.68.1-1.2.44-1.56 1.02-.22.38-.33.78-.33 1.2 0 .1.01.2.03.3v.03l1.38 9.33c.08.54.34.99.78 1.35a2.27 2.27 0 0 0 1.77.45c.66-.1 1.17-.42 1.53-.96.26-.36.39-.77.39-1.23 0-.08-.01-.17-.03-.27l-1.38-9.33c-.08-.56-.34-1.03-.78-1.41-.42-.34-.9-.51-1.44-.51-.1 0-.2.01-.3.03Zm18.78-.03c-.54 0-1.02.18-1.44.54-.44.36-.7.82-.78 1.38l-1.41 9.51c-.02.08-.03.15-.03.21 0 .52.17.98.51 1.38.36.46.84.73 1.44.81.1.02.21.03.33.03.66 0 1.21-.24 1.65-.72.3-.32.49-.7.57-1.14v-.06l1.38-9.36v-.03c.02-.1.03-.2.03-.3 0-.42-.12-.82-.36-1.2-.36-.58-.87-.92-1.53-1.02h-.06c-.1-.02-.2-.03-.3-.03Zm-9.24 8.04h-.06c-.08 0-.16.01-.24.03-.36.04-.69.16-.99.36-.16.1-.3.22-.42.36-.08.08-.21.22-.39.42l-2.19 2.7c-.18.2-.42.48-.72.84-.28.34-.51.59-.69.75-.38.34-.75.58-1.11.72-.2.08-.41.14-.63.18-.14.02-.29.03-.45.03-.38 0-.74-.07-1.08-.21-.32-.12-.6-.29-.84-.51l.75 5.13v.06c.02.04.03.09.03.15.04.18.09.35.15.51.14.3.35.56.63.78.18.16.38.28.6.36.24.08.49.12.75.12h.09c.16 0 .3-.01.42-.03.46-.1.85-.32 1.17-.66l.3-.3 4.89-6.03 4.74 5.82c.24.28.39.45.45.51.38.38.83.61 1.35.69h.33c.28 0 .55-.05.81-.15.42-.18.77-.46 1.05-.84.18-.28.3-.59.36-.93v-.06l.75-5.01c-.34.3-.73.5-1.17.6-.26.08-.52.12-.78.12a2.903 2.903 0 0 1-1.26-.27c-.26-.12-.59-.37-.99-.75-.26-.26-.59-.65-.99-1.17l-2.52-3.06c-.14-.18-.28-.34-.42-.48-.2-.2-.39-.35-.57-.45-.3-.2-.64-.31-1.02-.33h-.09Zm19.806.3c.28-.26.6-.46.96-.6.34-.14.69-.21 1.05-.21.46 0 .9.11 1.32.33.48.24.87.58 1.17 1.02a5.912 5.912 0 0 1 2.13-3.15 5.73 5.73 0 0 1 3.6-1.23c1.66 0 3.07.59 4.23 1.77 1.16 1.16 1.74 2.57 1.74 4.23 0 1.66-.58 3.08-1.74 4.26-1.16 1.16-2.57 1.74-4.23 1.74-1.34 0-2.55-.4-3.63-1.2a5.913 5.913 0 0 1-2.13-3.18c-.22.34-.5.63-.84.87-.5.3-1.04.45-1.62.45-.42 0-.83-.08-1.23-.24-.3-.14-.56-.32-.78-.54.5 2.42 1.71 4.42 3.63 6 1.92 1.56 4.12 2.34 6.6 2.34 1.9 0 3.65-.47 5.25-1.41 1.62-.94 2.89-2.21 3.81-3.81.94-1.62 1.41-3.38 1.41-5.28 0-1.9-.47-3.65-1.41-5.25-.92-1.62-2.19-2.9-3.81-3.84-1.6-.94-3.35-1.41-5.25-1.41-2.46 0-4.65.79-6.57 2.37-1.92 1.56-3.14 3.55-3.66 5.97Zm2.01-.09c-.66 0-1.21.24-1.65.72-.24.24-.41.53-.51.87-.06.22-.09.45-.09.69 0 .22.03.43.09.63.08.28.21.53.39.75.44.56 1.03.84 1.77.84.72 0 1.3-.27 1.74-.81.22-.26.37-.54.45-.84.04-.2.06-.4.06-.6 0-.18-.02-.35-.06-.51-.08-.38-.24-.71-.48-.99a2.23 2.23 0 0 0-1.71-.75Zm29.649-8.25c-.6 0-1.12.21-1.56.63-.46.44-.69.98-.69 1.62v6.15c.24-.28.53-.5.87-.66.42-.2.87-.3 1.35-.3h.06c.46 0 .9.1 1.32.3a2.909 2.909 0 0 1 1.05.84l9.81 11.55c.44.58 1.03.87 1.77.87.66 0 1.22-.25 1.68-.75.38-.42.57-.92.57-1.5v-6.12c-.26.26-.56.47-.9.63-.42.2-.87.3-1.35.3-.56 0-1.08-.13-1.56-.39-.34-.2-.63-.45-.87-.75l-9.87-11.67c-.22-.24-.48-.43-.78-.57a2.26 2.26 0 0 0-.9-.18Zm13.98 0c-.62 0-1.14.21-1.56.63-.46.44-.69.98-.69 1.62v8.7c0 .42.14.8.42 1.14.44.56 1.05.84 1.83.84.54 0 1.02-.15 1.44-.45.52-.4.79-.91.81-1.53v-8.7c-.02-.7-.29-1.27-.81-1.71-.42-.36-.9-.54-1.44-.54Zm-14.01 8.07c-.6 0-1.11.19-1.53.57-.44.4-.67.88-.69 1.44v8.67c0 .5.15.95.45 1.35.46.6 1.06.9 1.8.9.64 0 1.18-.23 1.62-.69.42-.42.63-.94.63-1.56v-8.67c-.02-.8-.41-1.39-1.17-1.77-.32-.16-.67-.24-1.05-.24h-.06Z"
            fill="currentColor"
          />
        </svg>
      </div>
      <div
        class="c24"
      >
        <div
          class="c25"
        >
          <h2
            class="c19 c26"
            color="black"
          >
            Contact
          </h2>
          <a
            href="mailto:contact@wongames.gg"
          >
            suporte@wongames.gg
          </a>
        </div>
        <div
          class="c25"
        >
          <h2
            class="c19 c26"
            color="black"
          >
            Follow us
          </h2>
          <nav
            aria-labelledby="social media"
          >
            <a
              href="#"
              rel="noopenner, noreferrer"
              target="-blank"
            >
              Instagram
            </a>
            <a
              href="#"
              rel="noopenner, noreferrer"
              target="-blank"
            >
              Twitter
            </a>
            <a
              href="#"
              rel="noopenner, noreferrer"
              target="-blank"
            >
              YouTube
            </a>
            <a
              href="#"
              rel="noopenner, noreferrer"
              target="-blank"
            >
              Facebook
            </a>
          </nav>
        </div>
        <div
          class="c25"
        >
          <h2
            class="c19 c26"
            color="black"
          >
            Links
          </h2>
          <nav
            aria-labelledby="social media"
          >
            <a
              href="#"
            >
              Loja
            </a>
            <a
              href="#"
            >
              Explorar
            </a>
            <a
              href="#"
            >
              Buscar
            </a>
            <a
              href="#"
            >
              FAQ
            </a>
          </nav>
        </div>
        <div
          class="c25"
        >
          <h2
            class="c19 c26"
            color="black"
          >
            Location
          </h2>
          <address>
            <span>
              Rua 7 de Maio
            </span>
            <span>
              527 - 89020330
            </span>
            <span>
              Rio de Janeiro, Brasil
            </span>
          </address>
        </div>
      </div>
      <div
        class="c27"
      >
        Won Games 2025 © Todos os Direitos Reservados
      </div>
    </footer>
  </div>
</section>
`;
